connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -69,6 +69,9 @@ class SparkSession private[sql] (
 
   private[this] val allocator = new RootAllocator()
 
+  // a unique session ID for this session from client.
+  private[sql] def sessionId: String = client.sessionId
```

Comment:
Nit: this to be more accurate should be `user_id+session_id`. However because session_id itself is a random number thus `client.sessionId` should still be able to identify SparkSession uniquely.

Reply:
https://github.com/apache/spark/blob/fa6e55bba4d62cdad66e5f425d8a261fe1050134/connector/connect/common/src/main/protobuf/spark/connect/base.proto#L48 this is the user_id of a client.

Reply:
Good advise, should I change it for this PR? @HyukjinKwon @hvanhovell

Reply:
I think current way is fine so I left it as a `Nit`.